code/dynamic_programming/src/rod_cutting/RodCutting.java

```diff
@@ -0,0 +1,31 @@
+/*  A Dynamic Programming solution for Rod cutting problem
+  Returns the best obtainable price for a rod of length a and
+   price[] as prices of different pieces */
+
+public class RodCutting {
+	
+	private static int cutRod(int price[] , int a)
+	{
+		int val[] = new int[a + 1];
+		val[0] = 0;
+
+		for (int i = 1; i <= a; ++i)
+		{
+			int max_val = Integer.MIN_VALUE;
+			for (int j = 0; j < i; ++j)
+				max_val = Math.max(max_val,price[j] + val[i-j-1]);
```

Comment:
Use consistent indentation here. Add spaces around the operators for `i - j - 1`.